OpenSim/Simulation/Model/TwoFrameLinker.h

```diff
@@ -80,6 +80,20 @@ class TwoFrameLinker : public C {
         const std::string& frame1Name,
         const std::string& frame2Name);
 
+    /** Convenience Constructor with two Frames
+    Construct a TwoFrameLinker where the two frames are specified by name
+    and offset transforms on the respective frames.
```

Comment:
This doxygen was copy-pasted and is incorrect; the frames are not specified by name.